### PR TITLE
Django models CRUD API testing

### PR DESCRIPTION
**Pull request by abhaykatheria**
This commit adds 'testApp' to the existing project and will be used for
creating further tests.

This commit also adds tests for creating, reading, updating, and
deleting Django models.

This is work towards vitessio/vitess#7905

Signed-off-by: Abhay Katheria <abhay.katheria1998@gmail.com>

### CODE REVIEW

**Review by mcronce — APPROVED** [review]
LGTM, thanks!

**Comment by abhaykatheria** [comment]
I think we can get rid of the teardown method as the CI test are failing.  Although they work fine on my local machine.



image

File: frameworks/python/django/src/testApp/test/test_model_crud/test_crud.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# These are testse
+
+from django.test import TestCase
+from testApp.models import Person
+
+class CrudTestCase(TestCase):
```
Comment (GuptaManan100) [inline frameworks/python/django/src/testApp/test/test_model_crud/test_crud.py:6]: Here could we also include the link to the guide that is tested as a comment.

**Review by GuptaManan100 — CHANGES_REQUESTED** [review]
Great work @abhaykatheria! 🎸

File: frameworks/python/django/src/testApp/test/test_model_crud/test_crud.py (hunk 1 above)
Comment (abhaykatheria) [reply frameworks/python/django/src/testApp/test/test_model_crud/test_crud.py:6]: I was planning on making a readme file for this directory explaining what is tested and how it is tested

File: frameworks/python/django/src/testApp/test/test_model_crud/test_crud.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# These are testse
```
Comment (GuptaManan100) [inline frameworks/python/django/src/testApp/test/test_model_crud/test_crud.py:1]: nit: Typing error

**Review by GuptaManan100 — APPROVED** [review]
LGTM too

**Pull request merged**